.pre-commit-config.yaml

```diff
@@ -12,6 +12,12 @@
 #
 # See https://github.com/pre-commit/pre-commit
 
+# If we enable pre-commit.ci, this will be useful:
+ci:
+  skip:
+  - docker-clang-format
+  # check-style maybe too?
+
```

Comment:
Why would you want to skip clang-format for pre-commit.ci? 
If I understand correctly, you are not yet using `pre-commit.ci`, so it doesn't yet matter.

Reply:
Because it runs in a docker container, so can't run docker. It doesn't have the correct version (or any version) of pre-commit preinstalled, the local one wouldn't help either.

No, we don't have it yet, but maybe someday. It would auto-update the pins, and apply Black changes automatically to PRs.